src/managers/k8s.py

```diff
@@ -15,13 +18,12 @@
 
 from literals import SECURITY_PROTOCOL_PORTS, AuthMap, AuthMechanism
 
-logger = logging.getLogger(__name__)
-
 # default logging from lightkube httpx requests is very noisy
-logging.getLogger("lightkube").disabled = True
-logging.getLogger("lightkube.core.client").disabled = True
-logging.getLogger("httpx").disabled = True
-logging.getLogger("httpcore").disabled = True
+logging.getLogger("lightkube").setLevel(logging.CRITICAL)
```

Comment:
Is lightkube noisy even at `ERROR` level?

Reply:
I could probably drop it to ERROR, but honestly I wanted to remove all of them, as it logs out the full HTTP request all the time, completely clogging the DEBUG logs